cmd/mattermost/commands/server.go

```diff
@@ -34,6 +36,25 @@ func init() {
 	RootCmd.RunE = serverCmdF
 }
 
+func loadCustomDefaults() (*model.Config, error) {
+	var customDefaults *model.Config
+	customDefaultsPath := os.Getenv("MM_CUSTOMDEFAULTS")
+
+	if customDefaultsPath != "" {
+		file, err := os.Open(customDefaultsPath)
+		if err != nil {
+			return nil, errors.Wrapf(err, "unable to load config custom defaults")
+		}
+		defer file.Close()
+
+		err = json.NewDecoder(file).Decode(&customDefaults)
+		if err != nil {
+			return nil, errors.Wrapf(err, "unable to decode custom defaults configuration")
+		}
+	}
+	return customDefaults, nil
```

Comment:
As the `customDefaults` variable is declared but not initialised, it has `nil` value if there are not custom defaults, so it would return `nil` in that case

Reply:
Yeah nvm I